Controllers/DishesController.js

```diff
@@ -45,5 +47,24 @@ const getAllDishesOfType = (req, res) => {
     })
 }
 
+const getOneDish = (req, res) => {
+    DbService.connectToDb(async (db) => {
+        const dishId = ObjectId(req.params.id)
+        try {
+            const dish = await DishesService.getOneDish(db, dishId)
+            let response = JSONResponseService.generateSuccessResponse()
+            response.message = "Requested dish retrieved successfully."
+            response.data = dish
+            res.json(response)
+            } catch (e) {
+                let response = JSONResponseService.generateFailureResponse()
+                response.message = "The resources requested do not exist at the desired location."
+                res.json(response)
```

Comment:
Just to be consistent with the rest of the application I suggest popping a return before the res.jsons